### DISCUSSION TITLE
Connecting to wifi

### DISCUSSION THREAD

**Discussion by Fuzzybee85** [CircuitSetup/Time-Circuits-Display#24] [General]
What do you do when the wifi doesn't show up from the time circuits on any device? It did when I first got the kit assembled but now it does not show

**Comment by CircuitSetup** [comment]
Was the TCD ever connected to local wifi? If so, your router should have given it an IP address.

**Comment by Fuzzybee85** [comment]
Yes I did connect it to my wifi a few months ago. So after that it won’t show the TCD wifi name anymore? How’s the best way to find the up address? And also I tried doing a firmware update to the 2.0.0 in arduino connected to pc last night but now when I plug in the time circuits nothing comes on the displays. Just the led lights come on. Can you update firmware like that or have to do it with wifi? On Oct 24, 2022, at 8:49 AM, John ***@***.***> wrote:﻿
Was the TCD ever connected to local wifi? If so, your router should have given it an IP address.
